examples/electron/package.json

```diff
@@ -33,6 +34,8 @@
   "devDependencies": {
     "@types/chai": "^3.5.1",
     "@types/webdriverio": "^4.7.0",
+    "@types/yargs": "^6.6.0",
```

Comment:
If `yargs` goes to `dependencies` then its type definitions should go there too. Rule; put them into the same place.